src/components/Editor/MonacoInstance/MonacoInstance.jsx

```
@@ -41,21 +43,45 @@ export const MonacoInstance = ({ }) => {
         if (content?.current) {
             editorRef.current.setValue(content.current);
         }
+        editorRef.current.layout();
     }
 
     // Editor options for Monaco Editor.
     const editorOptions = {
         scrollBeyondLastLine: false,
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)